test/lightning_web/live/work_order_live_test.exs

```diff
@@ -74,6 +74,17 @@ defmodule LightningWeb.RunWorkOrderTest do
                id: work_order.id,
                work_order: work_order
              ) =~ work_order.dataclip_id
+
+      Lightning.Repo.delete!(job)
+
+      work_order =
+        Lightning.Repo.reload!(work_order)
+        |> Lightning.Repo.preload([:attempts, :workflow])
+
+      assert render_component(LightningWeb.RunLive.WorkOrderComponent,
```

Comment:
Testing manually I see that the run for the deleted job is not list anymore. Asserting it would be a nice complement to the coverage.

Reply:
Thanks, I improved a bit the tests for more coverage.